tests/release_tests/test_launch/conftest.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import botocore
+
+LAUNCH_SANDBOX_ACCOUNT_NO = "305054156030"
```

Comment:
We don't want this to be public. This should be stored in a config file locally

Reply:
This constant is only used to confirm the caller is logged into the correct account. If the caller has to store it on their end, that sort of defeats the purpose since by that point they know to be logged in anyways. I can change it to check whether the caller is logged in and just not check the account number, which is better than nothing.

Reply:
Let's do that, I don't love the idea of having an account id available in our public repo